app/js/containers/Seller/5_SellerContactContainer.jsx

```diff
@@ -14,9 +16,11 @@ class SellerContactContainer extends Component {
       statusContactCode: props.seller.statusContactCode,
       ready: false
     };
-    props.footer.enableNext();
+    this.validate(props.seller.username, props.seller.statusContactCode);
     props.footer.onPageChange(() => {
       props.setContactInfo({username: this.state.username, statusContactCode: this.state.statusContactCode});
+    });
+    props.footer.onNext(() => {
       props.addOffer({...this.props.seller, username: this.state.username, statusContactCode: this.state.statusContactCode});
```

Comment:
Do you know I couldn't do:
`this.setState({isAddingOffer: true})`
in this callback function?